### ISSUE TITLE
Error "Use of undeclared identifier" in Xcode

### ISSUE THREAD

**Issue by sunshinewithmoonlight** [tanersener/ffmpeg-kit-test#23]
I'm sorry I am new to Xcode and not familiar with OC. I encountered an error after the following operation:

- In terminal
~~~
git clone https://github.com/tanersener/ffmpeg-kit-test
cd ffmpeg-kit-test
git checkout tags/ios.v4.5.1
~~~
- Open the `ios/test-app-local-dependency` project with Xcode ( Version 13.2.1 (13C100) )
- Change the `Development Team` options
- Change the `Product Bundle Identifier` options
- Drag folders in the the folder `ffmpeg-kit/prebuilt/apple-ios-arm64/ffmpeg/framework` into the framework directory of project and delete the original files with the same names
- Rename `ffmpeg-kit/prebuilt/apple-ios-arm64/ffmpeg-kit/include` to `ffmpeg-kit/prebuilt/apple-ios-arm64/ffmpeg-kit/ffmpegkit`, and drag folder `ffmpeg-kit/prebuilt/apple-ios-arm64/ffmpeg-kit` into option `System Header Search Path`
- Try to build on my iPhone 6s ( iOS 15.2 )
- It pops up a bunch of errors:
~~~
/Users/shine/Documents/git/ffmpeg-kit-test/ios/test-app-local-dependency/FFmpegKitIOS/FFmpegKitTest.m:152:46: ⚠️This old-style function definition is not preceded by a prototype
...
/Users/shine/Documents/git/ffmpeg-kit-test/ios/test-app-local-dependency/FFmpegKitIOS/FFmpegKitTest.m:197:44: 🔴Use of undeclared identifier 'MediaInformationSession'
/Users/shine/Documents/git/ffmpeg-kit-test/ios/test-app-local-dependency/FFmpegKitIOS/FFmpegKitTest.m:197:30: 🔴Use of undeclared identifier 'sessions3'
...
~~~

Why do errors occur?

**Comment by tanersener** [comment]
You're copying the wrong files. You must copy the bundles. See the following scripts. We use them to for `test-app-local-dependency`.

https://github.com/tanersener/ffmpeg-kit-test/blob/main/scripts/update_ios_local_dependency_xcframeworks.sh#L8
https://github.com/tanersener/ffmpeg-kit-test/blob/main/scripts/update_ios_local_dependency_frameworks.sh#L8
https://github.com/tanersener/ffmpeg-kit-test/blob/main/scripts/update_ios_local_dependency_frameworks_lts.sh#L8

**Comment by sunshinewithmoonlight** [comment]
Because I got an error using command `ffmpeg-kit/apple.sh`, I had to use command `ffmpeg-kit/ios.sh`. This results in me having no XCFramework folder and only framework folder, but I don't think is a big problem. I manually deleted the invalid xcframework folders in the project, and copied the `*.framework` folders from `ffmpeg-kit/prebuilt/bundle-apple-framework-ios` into the `framework` folder of the project. The above question disappears. I sincerely appreciate your help.

I'm now running into another problem, compiler prompts:

~~~
Command PhaseScriptExecution failed with a nonzero exit code
~~~
Then here is the detail of the error:

`/Users/shine/Library/Developer/Xcode/DerivedData/FFmpegKitIOS-bdwxpktjllcjsxcyijmotdpavnri/Build/Intermediates.noindex/FFmpegKitIOS.build/Debug-iphoneos/FFmpegKitIOS.build/Script-3448266727328FD400ECED5A.sh:`
`line 3: /Users/shine/Library/Developer/Xcode/DerivedData/FFmpegKitIOS-bdwxpktjllcjsxcyijmotdpavnri/Build/Products/Debug-iphoneos/FFmpegKitIOS.app/Frameworks/ffmpegkit.framework/Resources/strip-frameworks.sh: No such file or directory`

I tried copying the file `ffmpeg-kit/tools/release/apple/strip-frameworks.sh` to the appropriate location. Well, the build succeeded, but the installation failed. The installation failed on both my iPad mini5 and iPhone6s.
